tests/models/llama/test_modeling_llama.py

```diff
@@ -739,6 +739,29 @@ def test_compile_static_cache(self):
         static_compiled_text = tokenizer.batch_decode(generated_ids, skip_special_tokens=True)
         self.assertEqual(EXPECTED_TEXT_COMPLETION[self.cuda_compute_capability_major_version], static_compiled_text)
 
+    @slow
+    @require_torch_gpu
+    @require_read_token
+    def test_quantized_cache(self):
```

Comment:
this would require quanto to run the tests, not sure what is the best strategy here, maybe we could add again quanto in the `transformers-all-latest-gpu` .. The other solution would be to transfer the test here: https://github.com/huggingface/transformers/blob/main/tests/quantization/quanto_integration/test_quanto.py I think transferring the test there would be better. cc @SunMarc FYI

Reply:
Yes definitely agree with @younesbelkada I think that we can either transfer it to test_quanto or even create a new file test_quanto_kv_cache.py in quanto_integration folder.